javascripts/govuk/multivariate-test.js

```diff
@@ -71,18 +71,16 @@
   };
 
   MultivariateTest.prototype.setCustomVar = function(cohort) {
-    window._gaq = window._gaq || [];
-    window._gaq.push([
-      '_setCustomVar',
-      this.customVarIndex,
-      this.cookieName(),
-      cohort,
+    GOVUK.analytics.setDimension(
+      this.customVarIndex, 
+      this.cookieName(), 
+      cohort, 
       2 // session level
-    ]);
+    ); 
     // Fire off a dummy event to set the custom var on the page.
     // Ideally we'd be able to call setCustomVar before trackPageview,
     // but would need reordering the existing GA code.
```

Comment:
There is no guarantee that it will be run before the initial pageview is tracked. In an ideal world I think ensuring the dimensions are pushed to Google could be abstracted away inside our `GOVUK.analytics` object so we don't need to worry about it in other modules.